hashicorp/web-unified-docs
# chore: copy over terraform-docs-common

### Description

[Asana task](https://app.asana.com/0/1207899860738460/1208471953509312/f)

This PR copies over the latest docs from [terraform-docs-common](https://github.com/hashicorp/terraform-docs-common). There have been quite a few changes and updates to the docs over the past few months which is why the PR is so large.

### Testing

1. Run UDR locally using this branch: `leah/chore/copy-terraform-docs-common`
2. Go to one of the new pages and verify it loads and has content. One example is http://localhost:3000/terraform/cloud-docs/projects
3. Verify that some of the content changes are reflected
